ml-notebook/environment.yml

```diff
@@ -7,3 +7,4 @@ dependencies:
  - cudatoolkit=10
  - tensorflow>=2
  - jax
+ - jaxlib
```

Comment:
That won't help without a pin to `jaxlib<0.3` (or maybe lower).

When https://github.com/conda-forge/tensorflow-feedstock/pull/215 passes you can remove the pin.